src/app/tables/table.component.ts

```diff
@@ -13,6 +13,9 @@ export class EstuaryTableComponent implements OnChanges {
   @Input() title: string;
   // An optional title link
   @Input() titleLink: string;
+  // Only necessary if there is no title ; the name of the CSV file defaults to the
```

Comment:
`defaults to the title name` is a little misleading since the name of the CSV file is derived from the title by default, but not used as is.